identity-provider-service/src/bin/main.rs

```diff
@@ -1798,4 +1926,55 @@ mod tests {
             }
         });
     }
+
+    #[test]
+    fn test_broken_endpoint() {
+        tokio_test::block_on(async {
+            let response = test::request()
+                .method("GET")
+                .reply(&warp::get().and_then(get_broken_reply))
+                .await;
+            // Then
+            assert_eq!(response.status(), StatusCode::BAD_REQUEST);
+        });
+    }
+
+    #[test]
+    fn test_retrieve_failed_identity_token_before_delay_until() {
+        tokio_test::block_on(async {
+            let response = test::request()
+                .method("GET")
+                .path(&format!(
+                    "/{}",
+                    chrono::offset::Utc::now().timestamp() + 100
+                ))
+                .reply(
+                    &warp::get()
+                        .and(warp::path!(i64))
+                        .and_then(retrieve_failed_identity_token),
+                )
+                .await;
+            // Then
```

Comment:
I don't quite get this comment. Is it meant to be here?

Reply:
I kinda copied it from one of the other tests, I removed it.